### PR TITLE
Bug 1391307 - Run main summary with all histograms

### PR DESCRIPTION
**Pull request by fbertsch**
A few related changes to the Databricks operator, paralleled to telemetry-batch-view.py for consistency.

### CODE REVIEW

**Comment by fbertsch** [comment]
This has been tested multiple times in bug 1461333, with outputs in `s3://telemetry-test-bucket/main_summary/v4/submission_date_s3={20181002-20181005}`.

**Comment by fbertsch** [comment]
@acmiyaguchi I switched to running on a new task in `telemetry-backfill`. We can test existing jobs from there

**Comment by acmiyaguchi** [comment]
@fbertsch There's still value in running the entire dag in the dev environment -- all of the jobs will eventually need to be moved over. It should be safe and relatively fast to run the main_summary dag.

**Comment by fbertsch** [comment]
@acmiyaguchi agreed, but this PR doesn't block on that now.

File: dags/main_summary.py [hunk 1]
```diff
@@ -23,6 +24,32 @@
...
+    enable_autoscale=True,
+    instance_type="c4.4xlarge",
+    spot_bid_price_percent=50,
+    ebs_volume_count=1,
+    ebs_volume_size=250,
+    env=tbv_envvar("com.mozilla.telemetry.views.MainSummaryView",
+        options={
+            "from": "{{ ds_nodash }}",
+            "to": "{{ ds_nodash }}",
+            "schema-report-location": "s3://{{ task.__class__.private_output_bucket }}/schema/main_summary/submission_date_s3={{ ds_nodash }}",
```
Comment (acmiyaguchi) [inline dags/main_summary.py]: What is this going to do, by the way?

**Pull request merged**